setup.py

```diff
@@ -168,9 +168,9 @@
     "sudachipy>=0.6.6",
     "sudachidict_core>=20220729",
     # TensorFlow pin. When changing this value, update examples/tensorflow/_tests_requirements.txt accordingly
-    "tensorflow-cpu>=2.4,<2.13",
-    "tensorflow>=2.4,<2.13",
-    "tensorflow-text<2.13",
+    "tensorflow-cpu>=2.6",
```

Comment:
No let's leave a maximum pin. I do not want any surprises.

Reply:
Deal, but I'm bumping the cap to `<2.14`, since 2.13 works fine!

Reply:
Sure!